spec/transaction_search_shared_examples.rb

```diff
@@ -11,8 +11,9 @@
       let(:params) { super().merge(account_owners: [3, 4]) }
 
       it "applies the parameter" do
-        expect(assigns(:order_details).where_values)
-          .to be_include("account_users.user_id in ('3','4')")
+        where_values = assigns(:order_details).where_values.reject { |v| v.is_a?(String) }
+        expect(where_values.map(&:to_sql))
+          .to include("`account_users`.`user_id` IN (3, 4)")
       end
```

Comment:
Checking the Arel level makes sense if it's not too much effort. I like that better than the regex or checking for one of two strings. I think the idea was to avoid needing to set up a bunch of orders in order to run the queries and check the results.